src/content/events/2023/hackathon-october-2023.mdx

```diff
@@ -2,7 +2,8 @@
 title: Hackathon - October 2023 (Barcelona)
 subtitle: A hybrid hackathon held in Barcelona and online
 type: hackathon
-start_announcement: '2023-07-13T12:00+02:00'
+announcment:
```

Comment:
@mashehu - typo here

Reply:
Any idea why this didn't throw an error? The collections config should have been upset, no?

Reply:
It's not a required parameter and it doesn't care about addition frontmatter data.

Reply:
:shakefist:

Reply:
fixed in #2226